docker-compose.yml

```diff
@@ -39,41 +39,112 @@ services:
           ]
         }
 
+  mock-api:
+    image: bbyars/mountebank
+    volumes:
+      - ./modules/mocks/:/config:ro
+    ports:
+      - target: 5555
+        published: 5555
+    entrypoint:
+      - mb
+      - --configfile
+      - /config/mountebank.json
+
+  api:
+    image: ndsp/api
+    build:
+      context: ./modules/api
+      dockerfile: ./Dockerfile
+    depends_on:
+      - database
+    ports:
+      - target: 5002
+        published: 5002
+    volumes:
+      - ~/.aspnet/https:/https:ro
+    environment:
+      ASPNETCORE_ENVIRONMENT: Development
+      ASPNETCORE_URLS: http://+:5002
+      # TODO refactor this into Docker secrets management?
+      ASPNETCORE_Kestrel__Certificates__Default__Password: ndsa
+      ASPNETCORE_Kestrel__Certificates__Default__Path: /https/aspnetapp.pfx
+      ConnectionStrings__DefaultConnection: Host=database;Database=postgres;Username=postgres;Include Error Detail=true;
+
 
   user-portal:
```

Comment:
You need to use SSL or Chrome refuses to forward cookies with `SameSite=None` because of security features ; wrestled with this for too long trying to make it work over plain HTTP.